server/controllers/newUser.js

```diff
@@ -0,0 +1,31 @@
+const AWS = require('aws-sdk');
+const { uploadAudioFile, uploadImageFile, s3 } = require('../s3');
+const models = require('../models/index');
+
+const newUser = async (req, res) => {
+  const data = req.body;
+  console.log(data)
+  const usersTable = process.env.NODE_ENV === 'test' ? 'temp_users' : 'users';
```

Comment:
This can now be removed, this logic is inside models. We no longer need to pass tables into the models just for testing purposes